README.md

```diff
@@ -156,6 +156,9 @@ work on developing integration flows.
 
 Read on how to develop integration artifacts with EI tooling [here](https://docs.wso2.com/display/EI640/Integration+Tutorials).
 
+#### Test Artifacts in the Enterprise Integrator
+WSO2 Enterprise Integrator allows you to execute unit tests for your synapse configurations by using [Synapse Unit Testing Framework](gh-docs/synapse-unit-testing-framework/synapse-unit-testing-framework.md). 
```

Comment:
Shall we update to have "unit tests against your integration artifacts by using"

Reply:
Edited